fastlane/Fastfile

```diff
@@ -239,7 +262,80 @@ end
       project_slug: 'simplenote-macos',
       dsym_path: dSYM_PATH,
     )
+  end
+
+  #############################################################################
+  # build_and_upload_app_store
+  # ---------------------------------------------------------------------------
+  # This lane builds the app and uploads it to the Mac App Store, optionally
+  # creating a GitHub release.
+  # ---------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_upload_app_store [create_github_release:<create github release>]
+  #
+  # Example:
+  # bundle exec fastlane build_and_upload_app_store
+  # bundle exec fastlane build_and_upload_app_store create_github_release:true
+  #############################################################################
+  lane :build_and_upload_app_store do | options |
+    # This allows code signing to work on CircleCI. It is skipped if this isn't
+    # running on CI.
+    #
+    # See https://circleci.com/docs/2.0/ios-codesigning/
+    setup_circle_ci
+
+    sync_code_signing(
+      type: 'appstore',
+      platform: 'macos',
+      # This Mac app also needs a Mac Installer Distribution certificate
+      additional_cert_types: ['mac_installer_distribution'],
+      # Will default to false, unless a readonly value is specified, or is
+      # running on CI, in which case it will always be true.
+      readonly: options[:readonly] || is_ci,
```

Comment:
```suggestion
      readonly: true,
```

Nit: IMHO it's preferable to be extra-extra-extra safe here to avoid weird code signing issues. Not a requirement tho

Reply:
Right! I remember we discussed about this... I just copy pasted from elsewhere and forgot to updated 😳